articles/container-instances/container-instances-tutorial-deploy-app.md

```diff
@@ -47,7 +47,11 @@ Container registry password:
 az acr credential show --name <acrName> --query "passwords[0].value"
 ```
 
-To deploy your container image from the container registry with a resource request of 1 CPU core and 1 GB of memory, run the following command. Replace `<acrLoginServer>` and `<acrPassword>` with the values you obtained from the previous two commands.
+To deploy your container image from the container registry with a
+resource request of 1 CPU core and 1 GB of memory, run the following
+command, [`az container create`][az-container-create]. Replace `<acrLoginServer>` and `<acrPassword>` with the
```

Comment:
I recommend changing this to "`run the [az container create][az-container-create] command.`" (drop the "following" bit).